database-emulator/javascript-quickstart/test/test.js

```diff
@@ -0,0 +1,115 @@
+let firebase = require("@firebase/testing");
+fs = require('fs');
```

Comment:
`const fs =`

Reply:
You're right!
![face-palm](https://media2.giphy.com/media/AjYsTtVxEEBPO/giphy.gif?cid=3640f6095bcf73397950375232a0da4c)